service/replication.go

```diff
@@ -278,6 +278,18 @@ func (s *service) CreateStorageProtectionGroup(ctx context.Context,
 	}, nil
 }
 
+// TODO: implement
+// DeleteLocalVolume deletes the backend volume on the storage array.
+func (s *service) DeleteLocalVolume(ctx context.Context,
+	req *csiext.DeleteLocalVolumeRequest) (*csiext.DeleteLocalVolumeResponse, error) {
+	ctx, log, _ := GetRunIDLog(ctx)
+
+	log.Info("!!! Deleting Remote Volume !!!")
```

Comment:
Change this line to not include !!! and to be more accurate. We're deleting local volume, now.